osu.Game.Rulesets.Osu/Mods/OsuModStrictTracking.cs

```diff
@@ -70,6 +70,11 @@ public void ApplyToDrawableRuleset(DrawableRuleset<OsuHitObject> drawableRuleset
             drawableRuleset.Playfield.RegisterPool<StrictTrackingSliderTailCircle, StrictTrackingDrawableSliderTail>(10, 100);
         }
 
+        private class StrictTrackingSliderTick : SliderTick
+        {
+            public override Judgement CreateJudgement() => new OsuIgnoreJudgement();
```

Comment:
Is there a reason for this? ie. why can't we use normal slider ticks instead?

Reply:
If you're ok with the ticks showing in hit statistics on the results screen, then I don't see why not make them normal ticks - but making them have ignore judgements made sense to me seeing that the ticks aren't the primary gameplay element ensuring that the slider is being followed when this mod is enabled. Which is why I proposed that to begin with.

Reply:
I don't know. It basically changes the scoring of a beatmap with very little explanation to the user. Unless there's a good reason to do this, I'd prefer not to. Like, mods should be as unobtrusive on the general flow of things as possible, IMO.

Reply:
I've checked locally how this would work, and using normal ticks looks to be working just fine, but I'm going to wait for @apollo-dw's say on this before I go pushing things in to make sure I haven't missed anything.

Reply:
Sure, let's do it if you think this is the correct direction